src/hooks/sdk/useSelectAudioOutputDevice.tsx

```diff
@@ -1,17 +1,23 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { DefaultBrowserBehavior } from 'amazon-chime-sdk-js';
 import { useCallback } from 'react';
 
 import { useMeetingManager } from '../../providers/MeetingProvider';
-import { supportsSetSinkId } from '../../utils/device-utils';
 
-export const useSelectAudioOutputDevice = () => {
+export const useSelectAudioOutputDevice = (): ((
+  deviceId: string
+) => Promise<void>) => {
   const meetingManager = useMeetingManager();
 
   const selectDevice = useCallback(async (deviceId: string) => {
-    if (supportsSetSinkId()) {
+    if (new DefaultBrowserBehavior().supportsSetSinkId()) {
       await meetingManager.selectAudioOutputDevice(deviceId);
+    } else {
+      console.error(
```

Comment:
Should we throw error instead?

Reply:
This change is just to add some more logs. Don't think we need to throw this error.

Reply:
Notice that `chooseAudioOutput` API will throw an error if browser does not support setSinkId. If so the error will be threw in `MeetingManger`.